### ISSUE TITLE
TTX2 I/O errors & crashes

### ISSUE THREAD

**Issue by rufotheone-fr** [teknogods/OpenParrot#76]
Those games need a fix to run:

- **Battle Fantasia** = I/O error
- **BlazBlue: Calamity Trigger** = I/O error
- **KOF Maximum Impact Regulation A** = A window opens and immediately shuts down.
- **Wacky Races** = A window opens and immediately shuts down.

**Comment by adamgp** [comment]
Regarding KOF MIRA, runs fine if D drive is present.

**Comment by Bunny-Head** [comment]
Hello!
Have been trying to run MIRA for a while. I have a D:\ "fake" letter mount (sustb). I can reach the logos but as soon as the game starts it freezes. Any ideas?

**Comment by marcosgaming10** [comment]
No, KOF Maximum Impact Regulation A  starts but stop on Black Screen. I try to press Insert Coin but nothing....
